c2rust-refactor/src/scripting.rs

```diff
@@ -97,6 +104,24 @@ pub fn run_lua_file(
     Ok(())
 }
 
+/// Takes a set of marks and turns it into a lua table
+fn lua_serialize_marks<'lua>(
```

Comment:
Should this be called `get_marks`? It would be nice to have a 1:1 mapping from Lua functions to their Rust implementations.

Reply:
Think so? It seemed like serializing was more literally what it's doing from rust's perspective, while "getting" them from Lua's

Reply:
I think `serialize_marks` is fine here, since it's not getting the marks out of the command state but taking them as input.